guide/README.md

```diff
@@ -369,6 +370,27 @@ block `Mi` is lowered by one half step; on the right, both *Pitch*
 blocks are raised by one half step. (You can also use a double-sharp
 or double-flat accidental.)
 
+
+#### <a name= "DOUBLE-SHARPS-AND-FLATS">3.2.2.1 Double Sharps And Flats</a>
+
+Double sharps and flats, although not very common, are used in music
+notation to accurately represent pitches that are altered by two
+semitones. They arise in compositions that explore chromaticism,
+modulations to distant keys, or non-traditional scales where such
+alterations are necessary for harmonic or melodic purposes. Double
+sharps and flats help maintain the correct intervals, ensuring
+precise notation and clarity of musical expression.
+
+![double sharp and flat](./transform1_1.png "Using Double Sharp and Flat blocks")
```

Comment:
In addition to using the "accidental override", I think you should add examples of selecting those accidentals from the pie menu as follows:

![Screenshot from 2024-04-22 16-00-19](https://github.com/sugarlabs/musicblocks/assets/13454579/dada9d90-10ea-4b0e-aeed-d2238004ffd4)
![Screenshot from 2024-04-22 16-01-18](https://github.com/sugarlabs/musicblocks/assets/13454579/0286b9d0-e76c-46f5-a7db-a5f38f7c91c3)
![Screenshot from 2024-04-22 16-01-32](https://github.com/sugarlabs/musicblocks/assets/13454579/8704ab74-c37d-41b0-ae2d-5fac8f7721e5)